okapi-core/src/main/java/org/folio/okapi/util/ProxyContext.java

```diff
@@ -25,6 +26,8 @@
 public class ProxyContext {
 
   private static final Logger logger = OkapiLogger.get();
```

Comment:
```suggestion
  private static final Logger logger = OkapiLogger.get();  // logger name "okapi"
```
This is different from log4j's default name: LoggerManager.getLogger() uses the caller class ProxyContext.